scripts/st2bootstrap-deb.sh

```diff
@@ -79,6 +79,10 @@ install_st2mistral() {
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
   # Register mistral actions
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
+  
+  # Start Mistral
+  sudo service mistral-server start
+  sudo service mistral-api start
 }
```

Comment:
> sudo apt-get install -y sysvinit-utils
may be check if it's not installed and complain? IMO not a priority, and not exactly relevant to this, so I go merge, and if you decide to make this change, go for it.